src/gui/kperfwidget.cpp

```diff
@@ -57,6 +57,16 @@ KPerfWidget::KPerfWidget(QWidget *parent) :
             });
         }
     }
+    muteDev = ui->muteDev;
+//    muteDev->setCurrentIndex(perf ? perf->getMuteDevice() : 1);
+    if (isMuteDeviceSupported())  {
+        muteDev->setEnabled(true);
+        connect(muteDev, QOverload<int>::of(&QComboBox::currentIndexChanged), [=] (int index) {
```

Comment:
We have a Q_OVERLOAD macro, please use that as it retains compatibility with older Qt versions that we still support.

Reply:
Sorry, it's called `OVERLOAD_PTR` https://github.com/ckb-next/ckb-next/blob/050231f2560fdfa25668cea1a57301dc79b944d4/src/gui/qoverloadlegacy.h